.github/workflows/ci.yml

```diff
@@ -562,7 +563,7 @@ jobs:
           kubectl get pulps.repo-manager.pulpproject.org
           sudo systemctl stop pulp-operator.service
           make local
-          sleep 5
+          sleep 10
```

Comment:
I had to increase the waiting time for the operator to update its status. Pending investigation on why we had to even wait for 5 seconds.